OneCommunityGlobal/HighestGoodNetworkApp
# Luis Fix project update speed on user profile and on time-log

# Description
<img width="767" alt="Screenshot 2023-11-05 at 7 26 53 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/25bf420e-4d8e-473a-9f18-eca77070e2c9">



## Related PRS (if any):
This frontend PR is related to the [#596](https://github.com/OneCommunityGlobal/HGNRest/pull/596)  backend PR.
…

## Main changes explained:
On the backend I added a check to remove cache. 
The issue was the backend was sending back old cache instead of clearing it when a project was added. 
Hence it was sending old data. 
The change occurs in `ProjectController.js `inside of the `assignProjectToUsers()` when mapping over each element cache is checked if so it removes it and adds the user. 


## How to test:
1. check into current branch
2. do `npm run install` and  then `npm run start` 
3. log as admin user
4. go to view-profile→ projects→ take note how many projects are there. 
5. Next go to other links → projects → select a project → add your current user that you are logged in as. 
6. Add the user then navigate  to the view-profile→ projects→ and check if the project was added to the list of projects.
7. Next click timelog check to see if the project is in the list of tasks.
8. click "add tangible time entry" 
9. click on projects/task and see if the newly added project is inside of the drop down menu

## Screenshots or videos of changes:



https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/96cb30a8-34a6-43d9-89a1-39998f196a21


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/59580642-563d-44a8-9a70-9091121d4c65



## Note:
Include the information the reviewers need to know.
